src/components/chat/comment-textbox.tsx

```diff
@@ -7,16 +7,22 @@ import "../themes.scss";
 interface IProps {
   activeNavTab?: string;
   numPostedComments: number;
-  onPostComment?: (comment: string) => void;
+  onPostComment?: (comment: string, tag: string) => void;
+  showCommentTag?: boolean;
+  commentTags?: Record<string, string>
```

Comment:
Looks like I have to do `Record<string, string> | never[];'  and also in `unit-configuration.ts` line 84 (see picture), otherwise it gives me an error. 
<img width="679" alt="image" src="https://github.com/concord-consortium/collaborative-learning/assets/521934/6400fa27-5c8b-45d6-9d15-289371aca737">